README.md

```diff
@@ -35,6 +35,7 @@ resources:
 * `target_branch`(string): Filter merge requests by target_branch. Default is empty string.
 * `source_branch`(string): Filter merge requests by source_branch. Default is empty string.
 * `sort` (string): Merge requests sorting order, either `asc` (default) or `desc` to reverse.
+* `recursive`: When set to `true`, will pull submodules.  If your submodules are relative, be sure to [use a relative path](https://www.gniibe.org/memo/software/git/using-submodule.html) to avoit https/git protocol clashing issues.
```

Comment:
```suggestion
* `recursive`: When set to `true`, will pull submodules.  If your submodules are relative, be sure to [use a relative path](https://www.gniibe.org/memo/software/git/using-submodule.html) to avoid https/git protocol clashing issues.
```

Note sure everyone will understand how it works as relative submodules are not that common. Maybe rephrase and be a bit more specific.

Reply:
Updated the README.md, might be better